docs/components/secnav/index.html

```diff
@@ -0,0 +1,138 @@
+            </a>
+  
+            <a href="#" class="selected">
+              <hx-icon type="ticketing"></hx-icon>
+              <p>Tickets</p>
+            </a>
+  
+            <a href="#">
+              <hx-icon type="support"></hx-icon>
+              <p>Support</p>
+            </a>
+  
+            <a href="#" class="disabled" id="billing">
+              <hx-icon type="billing"></hx-icon>
+              <p>Billing</p>
+              <hx-tooltip for="billing" position="bottom-center">
+                You do not have access to this area. Contact an account admin in your organization to request access to
+                this item.
+              </hx-tooltip>
+            </a>
+  
+            <a href="#">
+              <hx-icon type="account"></hx-icon>
+              <p>Account</p>
+            </a>
+          </a>
+          <div class="straightLine"></div>
+          <div>
+            <hx-disclosure id="topnav-menu" aria-controls="topnavMenuId" role="button" aria-expanded="false"
+              class="disabled">
+              <span>Jane User</span>
+              <hx-icon class="hxPrimary" type="angle-down"></hx-icon>
+            </hx-disclosure>
+            <hx-menu id="topnavMenuId" position="bottom-end">
+              <div class="sectionHeader">
+                <hx-menuitem role="menuitem" class="menuKey">Account Number :</hx-menuitem>
+                <hx-menuitem role="menuitem" class="menuValue">12345678</hx-menuitem>
+              </div>
+              <hr class="divider">
+              <div class="sectionBody">
+                <hx-menuitem role="menuitem" class="menuValue">My Profile & Settings</hx-menuitem>
+              </div>
+              <hr class="divider">
+              <div class="sectionFooter">
+                <button class="hxBtn">Log Out </button>
+              </div>
+            </hx-menu>
+          </div>
+        </div>
+      </div>
+      <!-- Secondary Navigation -->
+      <div id="hxSecNav">
+        <a href="#" class="navLink"> Nav Item One</a>
+        <div class="productMenu">
+          <hx-disclosure id="topnav-productMenu" aria-controls="secNavProductDropdown" role="button" aria-expanded="false"
+            class="disabled">
+            <span>Nav Item Two</span>
+            <hx-icon class="hxPrimary" type="angle-down"></hx-icon>
+          </hx-disclosure>
+          <hx-menu id="secNavProductDropdown">
+            <div class="sectionHeader">
+              <hx-menuitem role="menuitem" class="menuKey">Cloud Server Backups:</hx-menuitem>
+              <hx-menuitem role="menuitem">Systems</hx-menuitem>
+              <hx-menuitem role="menuitem">Backups</hx-menuitem>
+              <hx-menuitem role="menuitem">Activity</hx-menuitem>
+            </div>
+          </hx-menu>
+          <hx-disclosure id="topnav-menu" aria-controls="secNavProductDropdownTwo" role="button" aria-expanded="false"
+            class="disabled">
+            <span>Nav Item Three</span>
```

Comment:
The text should be all uppercase:

<img width="262" alt="Screen Shot 2020-10-14 at 2 43 42 PM" src="https://user-images.githubusercontent.com/10120600/96037730-d384a400-0e2b-11eb-9233-3378ddee1b23.png">

Reply:
Changes have been done, please review.